src/test/java/edu/hm/hafner/analysis/KotlinPackageDetectorTest.java

```diff
@@ -0,0 +1,54 @@
+package edu.hm.hafner.analysis;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import edu.hm.hafner.util.ResourceTest;
+
+import static java.nio.charset.StandardCharsets.*;
+import static org.assertj.core.api.Assertions.*;
+
+/**
+ * Tests the class {@link KotlinPackageDetector}.
+ *
+ * @author Bastian Kersting
+ */
+class KotlinPackageDetectorTest extends ResourceTest {
+    @ParameterizedTest(name = "{index} => file={0}, expected package={1}")
+    @CsvSource({
+            "MavenKotlinTest.txt, edu.hm.kersting",
+            "complicated-package-declaration-Kotlin.txt, edu.hm.kersting",
```

Comment:
Seems you are using a Windows machine, the test fails on Unix:
 
```suggestion
            "complicated-package-declaration-kotlin.txt, edu.hm.kersting",
```

Reply:
Fixed with ae1a39978dcc6a80d2951d0a012bb89a1d96e976